src/crypto/crypto_util.h

```diff
@@ -431,7 +431,11 @@ class CryptoJob : public AsyncWrap, public ThreadPoolWork {
 
     v8::Local<v8::Value> ret[2];
     env->PrintSyncTrace();
+    TRACE_EVENT_BEGIN(TRACING_CATEGORY_NODE2(crypto, sync),
+                      CryptoJobTraits::JobName);
```

Comment:
Is this purely experimental or is there any stability expectation, now or in the future? Whether any particular operation is implemented as a `CryptoJob` is an implementation detail, and not all sync crypto operations use this class.